query/benchmarker_test.go

```diff
@@ -36,8 +38,12 @@ func TestProcessorHandler(t *testing.T) {
 	var wg sync.WaitGroup
 	qPool := &testQueryPool
 	wg.Add(2)
-	go b.processorHandler(&wg, qPool, p1, 0)
-	go b.processorHandler(&wg, qPool, p2, 5)
+	var requestRate = rate.Limit(math.MaxFloat64)
```

Comment:
maybe a helper function called `getInfLimiter` since this is used in a couple of test functions